### PR TITLE
Formatted all files.

### PR DESCRIPTION
**Pull request by RoosterDragon**
Figured since everything was just StyleCop'd - we might as well round things off and unify the formatting a bit. I mostly let VS just autoformat everything, but I did manually fix up a few files.

### CODE REVIEW

**Comment by RoosterDragon** [comment]
[Diff ignoring whitespace](https://github.com/OpenRA/OpenRA/pull/7275/files?w=1)

**Comment by LipkeGu** [comment]
overlooked: 
https://github.com/RoosterDragon/OpenRA/blob/da5afd70a1e99c33ab64349cc3e526c112f0ec5b/OpenRA.Game/Server/Server.cs#L665

**Comment by RoosterDragon** [comment]
Not sure what you're getting at there - what am I missing?

File: OpenRA.Mods.RA/Traits/TargetableSubmarine.cs [hunk 1]
```diff
@@ -32,8 +32,11 @@ public TargetableSubmarine(Actor self, TargetableSubmarineInfo info)
 
 		public override string[] TargetTypes
 		{
-			get { return cloak.Cloaked ? info.CloakedTargetTypes
-									   : info.TargetTypes; }
+			get
+			{
+				return cloak.Cloaked ? info.CloakedTargetTypes
+									 : info.TargetTypes;
```
Comment (pchote) [inline OpenRA.Mods.RA/Traits/TargetableSubmarine.cs]: This is still a bit wonky.

**Comment by pchote** [comment]
Looks good to me, but needs a rebase.  We should aim to get this merged asap to avoid additional rebase conflicts.

File: OpenRA.Mods.RA/Traits/TargetableSubmarine.cs (hunk 1 above)
Comment (RoosterDragon) [inline OpenRA.Mods.RA/Traits/TargetableSubmarine.cs]: Done.

**Comment by RoosterDragon** [comment]
Rebased.

**Pull request merged**